further updated mbx files

davidfarmer opened the pull request:
These mbx files should address some of the items mentioned in recent issue discussions.

There are merge conflicts.  Not sure why, unless you changed mbx files?

Also, we need a .gitignore with the html files because there are a huge number of them and it
is too early in the process to start tracking them.

If necessary I can make a fresh repository with the mbx files, since at this point we only
want the output of SL2X.

oscarlevin commented:
Hi David,

I don't really understand what is going on here.  I have added *.html to .gitignore (in both master and develop) and removed all the html files from my branches.  Would that not also remove them from your copy when you pull?

I did not mean to commit it, but I was messing around with one of the mbx files.  That might be where the conflict is, but I'm not sure how to fix it.  Would it work if I remove all the mbx files from the repository (until you add them back)?

I don't understand why the commits listed in the pull request go all the way back to March 22.  They did last time too, so why would they do so again?

Also, could you target the develop branch instead of the master branch for the pull request?  Perhaps this will help.

davidfarmer commented:
I don't know how to recover from this.

I am going to set up a new repository *only* for the mbx files.

You can fork and pull from that repository, and then copy those
files into your "official" repository.

I'll set this up (hopefully today) and then email you.

This should avoid any further headaches.

On Wed, 6 Jul 2016, Oscar Levin wrote:

> 
> Hi David,
> 
> I don't really understand what is going on here. I have added *.html to .gitignore (in both master
> and develop) and removed all the html files from my branches. Would that not also remove them from
> your copy when you pull?
> 
> I did not mean to commit it, but I was messing around with one of the mbx files. That might be where
> the conflict is, but I'm not sure how to fix it. Would it work if I remove all the mbx files from the
> repository (until you add them back)?
> 
> I don't understand why the commits listed in the pull request go all the way back to March 22. They
> did last time too, so why would they do so again?
> 
> Also, could you target the develop branch instead of the master branch for the pull request? Perhaps
> this will help.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the
> thread.[AAM6LJ1hNZ4NQNn_dQozT0jiuE1cDbhQks5qTBi6gaJpZM4JGUMO.gif]
> 
> 
>

oscarlevin commented:
Sounds good.  Thanks for all your work on this.




Sent from my Sprint Samsung Galaxy S7 edge.


-------- Original message --------
From: davidfarmer <notifications@github.com>
Date: 7/6/16 4:28 PM (GMT-07:00)
To: oscarlevin/discrete-book <discrete-book@noreply.github.com>
Cc: "Levin, Oscar" <Oscar.Levin@unco.edu>, Comment <comment@noreply.github.com>
Subject: Re: [oscarlevin/discrete-book] further updated mbx files (#28)


I don't know how to recover from this.

I am going to set up a new repository *only* for the mbx files.

You can fork and pull from that repository, and then copy those
files into your "official" repository.

I'll set this up (hopefully today) and then email you.

This should avoid any further headaches.

On Wed, 6 Jul 2016, Oscar Levin wrote:

>
> Hi David,
>
> I don't really understand what is going on here. I have added *.html to .gitignore (in both master
> and develop) and removed all the html files from my branches. Would that not also remove them from
> your copy when you pull?
>
> I did not mean to commit it, but I was messing around with one of the mbx files. That might be where
> the conflict is, but I'm not sure how to fix it. Would it work if I remove all the mbx files from the
> repository (until you add them back)?
>
> I don't understand why the commits listed in the pull request go all the way back to March 22. They
> did last time too, so why would they do so again?
>
> Also, could you target the develop branch instead of the master branch for the pull request? Perhaps
> this will help.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the
> thread.[AAM6LJ1hNZ4NQNn_dQozT0jiuE1cDbhQks5qTBi6gaJpZM4JGUMO.gif]
>
>
>

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/oscarlevin/discrete-book/pull/28#issuecomment-230926389>, or mute the thread<https://github.com/notifications/unsubscribe/AGNAlIYGmpFcU8l3WDDLEK4EiqSgQ0VRks5qTCwWgaJpZM4JGUMO>.
